service/src/event/service.rs

```diff
@@ -129,23 +129,72 @@ impl CeramicEventService {
         )?)
     }
 
-    pub(crate) async fn insert_events(
-        &self,
-        items: &[recon::ReconItem<EventId>],
-        source: DeliverableRequirement,
-    ) -> Result<InsertResult> {
-        let mut to_insert = Vec::new();
-        let mut invalid = Vec::new();
+    async fn validate_signed_events(
+        events: Vec<EventInsertable>,
+    ) -> Result<(Vec<EventInsertable>, Vec<InvalidItem>)> {
+        // TODO: IMPLEMENT THIS
+        Ok((events, Vec::new()))
```

Comment:
should this be `unimplemented!();` or `todo!()`

Reply:
That will cause things to start breaking since it's not behind a feature flag yet.  I'm not too worried about us forgetting to do this 😅